src/N3Store.js

```diff
@@ -341,7 +339,7 @@ export default class N3Store {
     const stream = new Readable({ objectMode: true });
 
     stream._read = () => {
-      for (const quad of this.getQuads(subject, predicate, object, graph))
+      for (const quad of this._yieldQuads(subject, predicate, object, graph))
```

Comment:
We _could_ have smarter stream buffering; since it is streaming, we don't need to add everything at once here now.

Reply:
Forgot that it was synchronous.